src/coreclr/src/jit/codegenlinear.cpp

```diff
@@ -1293,6 +1308,69 @@ void CodeGen::genCheckConsumeNode(GenTree* const node)
 }
 #endif // DEBUG
 
+//--------------------------------------------------------------------
+// genConsumeReg: Do liveness update for a single register of a multireg child node
+//                that is being consumed by codegen.
+//
+// Arguments:
+//    tree          - GenTree node
+//    multiRegIndex - The index of the register to be consumed
+//
+// Return Value:
+//    Returns the reg number for the given multiRegIndex.
+//
+regNumber CodeGen::genConsumeReg(GenTree* tree, unsigned multiRegIndex)
+{
+    if (tree->OperGet() == GT_COPY)
```

Comment:
off-topic: I prefer `OperIs(GT_COPY)` because it is usually shorter and doesn't not need additional brackets in && conditions.